static/js/tools/stat_var/page.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import axios from "axios";
+import React, { Component } from "react";
+import { Explorer } from "./explorer";
+import { Info } from "./info";
+import { StatVarHierarchyType, StatVarSummary } from "../../shared/types";
+import { StatVarHierarchy } from "../../stat_var_hierarchy/stat_var_hierarchy";
+
+interface PageStateType {
+  displayName: string;
+  statVar: string;
+  summary: StatVarSummary;
+}
+
+class Page extends Component<unknown, PageStateType> {
+  constructor(props: unknown) {
+    super(props);
+    this.state = {
+      displayName: "",
+      statVar: "",
+      summary: { placeTypeSummary: {} },
+    };
+    this.fetchSummary();
```

Comment:
I think "this.fetchSummary()" here should be moved to "componentDidMount"

Reply:
done